corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -486,7 +486,7 @@ def _migrate_form_attachments(sql_form, couch_form):
             content_length=blob.content_length,
             blob_id=blob.key,
             blob_bucket="",  # temporary/special value -> new blob db API
-            md5=blob.info.md5_hash,
+            md5=blob.info.md5_hash or "unknown-md5",
```

Comment:
Just checking, why is this stub/constant value okay here to pass?

Reply:
The new blob db API does not use or maintain the MD5 hash, so in places where we don't have one and the value cannot be `None`, I used a value with obvious meaning if someone happens to read it.